dimelo/load_processed.py

```diff
@@ -34,39 +46,47 @@ def pileup_counts_from_bedmethyl(
     valid_base_count = 0
     modified_base_count = 0
 
-    mod_motif = motif.split(",")[0]
-    mod_coord_in_motif = motif.split(",")[1]
+    parsed_motif = utils.ParsedMotif(motif)
+    if parsed_motif.warning:
```

Comment:
I think this pattern is confusing. In the scenario where we implement logging, for example, I'd expect the warning to be printed by the ParsedMotif object itself. I think that can just be done here with printing anyway. I'll add a similar comment to the class itself.